requirements.txt

```diff
@@ -1,3 +1,4 @@
+-e git://github.com/leonardt/Pyverilog.git#egg=pyverilog
```

Comment:
do we need this?

Reply:
yes, see the top comment of the pull request

Reply:
👍